src/nemos/utils.py

```diff
@@ -459,6 +460,32 @@ def assert_scalar_func(func: Callable, inputs: List[jnp.ndarray], func_name: str
         )
 
 
+def format_repr(
+    obj: object, exclude_keys: List[str] = [], use_name_keys: List[str] = []
```

Comment:
```suggestion
    obj: Base, exclude_keys: List[str] = [], use_name_keys: List[str] = []
```

and add `from .base_class import Base`.